FreeRTOS/FreeRTOS-Plus-TCP
# Updates ESP32 network interface /multi

Description
-----------
This PR make the ESP32 driver compatible with /multi and IPv6.
All adapted drivers can still be used in backward compatible mode when `ipconfigCOMPATIBLE_WITH_SINGLE` is defined in FreeRTOSIPConfig.h.

Test Steps
-----------
Replace all /single sources with this branch, add FreeRTOS_Routing.c, define `ipconfigCOMPATIBLE_WITH_SINGLE` and run.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
